src/main/java/net/minecraftforge/registries/ForgeRegistries.java

```diff
@@ -129,7 +129,7 @@
 
         //Forge
         public static final ResourceKey<Registry<DataSerializerEntry>> DATA_SERIALIZERS = key("data_serializers");
-        public static final ResourceKey<Registry<GlobalLootModifierSerializer<?>>> LOOT_MODIFIER_SERIALIZERS = key("forge:loot_modifier_serializers");
+        public static final ResourceKey<Registry<DataSerializerEntry>> LOOT_MODIFIER_SERIALIZERS = key("forge:loot_modifier_serializers");
```

Comment:
Is this an intended change? It is hard to tell without being able to browse the code.

Reply:
No, this was likely a dumb IntelliJ refactor thing I missed. Resolved.